src/openms/include/OpenMS/FILTERING/DATAREDUCTION/Deisotoper.h

```diff
@@ -64,6 +64,9 @@ class OPENMS_DLLAPI Deisotoper
    * @param [max_isopeaks] The maximum number of isotopic peaks (at least 2) considered for an isotopic cluster
    * @param [make_single_charged] Convert deisotoped monoisotopic peak to single charge
    * @param [annotate_charge] Annotate the charge to the peaks in the IntegerDataArray: "charge" (0 for unknown charge)
+   * @param [annotate_iso_peak_count] Annotate the number of isotopic peaks in a pattern for each monoisotopic peak in the IntegerDataArray: "iso_peak_count"
+   * @param [use_averagine] Use a simple averagine model that expects heavier isotopes to have less intensity. If false, no intensity checks are applied.
```

Comment:
I would not call that averagine since it really is a much simpler model that assumes you have less than 60 or so Carbon molecules in your analyte. I would argue that using averagine here is actually misleading, maybe lets try `use_decreasing_model` or something that is more explicit